gradoop-store/gradoop-hbase/src/main/java/org/gradoop/storage/impl/hbase/io/HBaseDataSink.java

```diff
@@ -82,18 +54,12 @@ public void write(GraphCollection graphCollection) throws IOException {
   }
 
   @Override
-  public void write(
-    LogicalGraph logicalGraph,
-    boolean overwrite
-  ) throws IOException {
-    write(getFlinkConfig().getGraphCollectionFactory().fromGraph(logicalGraph), overwrite);
+  public void write(LogicalGraph logicalGraph, boolean overwrite) throws IOException {
+    write(getHBaseConfig().getGraphCollectionFactory().fromGraph(logicalGraph), overwrite);
   }
 
   @Override
-  public void write(
-    GraphCollection graphCollection,
-    boolean overWrite
-  ) throws IOException {
+  public void write(GraphCollection graphCollection, boolean overWrite) throws IOException {
 
     // TODO: interpret overWrite
```

Comment:
+ Open TODO. Probably for another PR or will you fix it?
+ Missing JavaDoc or inheritence hint

Reply:
ATM it is not planned to implement a overwrite strategy for graphs stored at HBase or Accumulo. I'll add a `Not implemented exception` if a function with overwrite was called, OK?